images/linux/ubuntu1604.json

```diff
@@ -166,6 +168,19 @@
             ],
             "execute_command": "sudo sh -c '{{ .Vars }} pwsh -f {{ .Path }}'"
         },
+        {
+            "type": "shell",
+            "scripts": [
+                "{{template_dir}}/scripts/installers/docker-compose.sh",
+                "{{template_dir}}/scripts/installers/docker-moby.sh"
```

Comment:
I intentionally moved these scripts into their own provisioner block so that the DockerHub credentials were not exposed to any other unrelated setup scripts.